server/.env.example

```diff
@@ -1,3 +1,8 @@
 # This will connect to the main branch of the db in the final changes
 # Rename this to .env and copy a dev branch url to here from PlanetScale 
-DATABASE_URL=''
+DATABASE_NAME='blueprint-tces'
```

Comment:
This file should be renamed to .env. ".env" is in the gitignore so that you don't accidentally push sensitive information such as the database username and password (Like here 😆)

Reply:
I have a .env file as well, I just copy/pasted my .env into here so that others can use the credentials but PlanetScale yelled at me for it 😆 . I'll keep everything except the username/password here as is so that others can access it easily